src/htm/algorithms/SpatialPooler.hpp

```diff
@@ -185,8 +185,9 @@ class SpatialPooler : public Serializable
         boost. Shorter values make it potentially more unstable and
         likely to oscillate.
 
-  @param boostStrength A number greater or equal than 0, used to
-        control boosting strength. No boosting is applied if it is set to 0.
+  @param boostStrength A number greater or equal than 1.0 or exactly 0.0, used to
```

Comment:
Please revert this change to the docs, since you reverted the corresponding code change.

Reply:
good catch, reverted. Thanks!